examples/Kittens/Sample/ViewController.m

```diff
@@ -96,12 +94,6 @@ - (NSMutableArray *)createLitterWithSize:(NSInteger)litterSize
   return kittens;
 }
 
-- (void)setKittenDataSource:(NSMutableArray *)kittenDataSource {
-  ASDisplayNodeAssert(!self.dataSourceLocked, @"Could not update data source when it is locked !");
```

Comment:
Haha, this is so old! Thanks for fixing. Long story about the locked data source mode of ASTableView...truly off-main edit operations :)